### PR TITLE
Feature/tao 4279 improve export

### PR DESCRIPTION
**Pull request by ssipasseuth**
This PR is a first part of TAO-4279.
It just help toggling the active/disable state for the submit button.
To test this, add a new items in an item class, uncheck all -> see results, check a few -> see results.

### CODE REVIEW

File: views/js/controller/export/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                }else{
+                    $formSubmitter.addClass('disabled');
+                }
+            };
+
+            updateSubmitter();
+            $container.on('change', ':checkbox', updateSubmitter);
+        }
+    };
+});
```
Comment (eacraft13) [inline views/js/controller/export/index.js]: This is an interesting use of a controller. Is there a reason this wasn't done in tao/views/templates/form/export.tpl? Placing this code here will cause headaches for developers updating the export form.

Comment (ssipasseuth) [inline views/js/controller/export/index.js]: nice catch, I didn't think about this one. ok will make the change

Comment (ssipasseuth) [inline views/js/controller/export/index.js]: so I made changes to https://github.com/oat-sa/tao-core/pull/1314 so this PR is no longer required

**Comment by ssipasseuth** [comment]
no longer needed

**Pull request closed without merging**